ui/src/app/views/breadcrumb/component.ts

```diff
@@ -86,13 +86,23 @@ export class BreadcrumbComponent implements OnInit, OnDestroy {
         label = child.snapshot.data['cohort'].name;
       }
       if (label === 'Param: Concept Sets Name') {
-        label = child.snapshot.data['workspace'].name + ' Concepts';
+        label = child.snapshot.data['workspace'].name;
         if (breadcrumbs.length > 2) {
           breadcrumbs = breadcrumbs.filter(b => !b.url.endsWith('/concepts'));
         }
       }
       if (label === 'Param: Concept Set Name') {
         label = child.snapshot.data['conceptSet'].name;
+        // For the most part, we don't want to append the current label to the breadcrumbs
+        //      since we want the label to stop at the parent.
+        // In this case, we want the breadcrumb header to be the Concept Set Name
+        const conceptSetBreadcrumb = BreadcrumbComponent.makeBreadcrumb(label, url, child);
```

Comment:
The existing route hierarchy of concepts defined in app-routing-module.ts doesn't necessarily make sense and I think can freely change in this PR, instead of adding logic here.

Probably what would make sense is:
- workspacePage
  - concepts search
  - concept sets list
    - concept set details